components/sysutils/coreutils/Makefile

```diff
@@ -46,21 +52,25 @@ PATCH_LEVEL=		0
 # Prevent gcc register spill error on 32-bit SPARC
 gcc_OPT.sparc.32 =	-O1
 
-CONFIGURE_PREFIX	 =	/usr/gnu
-CONFIGURE_OPTIONS	+=	--libdir=/usr/lib
+# Recreate configure and Makefile to pick up changes in .../src/local.mk
+COMPONENT_PREP_ACTION = (cd $(@D) ; autoreconf -f && automake)
+
+CONFIGURE_PREFIX	 =	$(GNUDIR)
+CONFIGURE_BINDIR.64	 =	$(GNUBIN)
+CONFIGURE_OPTIONS	+=	CPPFLAGS=" -I$(USRINCDIR)/gmp"
+CONFIGURE_OPTIONS	+=	--libdir=$(USRLIBDIR.$(BITS))
```

Comment:
solaris-userland builds both versions and delivers 
```
file $(MACH64)/src/libstdbuf.so path=usr/lib/$(MACH64)/libstdbuf.so
file $(MACH32)/src/libstdbuf.so path=usr/lib/libstdbuf.so
```
Perhaps, it's worth asking.